src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age > 0 && <p className="Person__age">{`I am ${person.age}`}</p>}
+    <p className="Person__partner">
+      {person.isMarried && (
+        <>
+          {`${person.partnerName} is my`}
+          {person.sex === 'f' ? ' husband' : ' wife'}
+        </>
+      )}
+
+      {!person.isMarried && `I am not married`}
```

Comment:
Make your code more easy to read with ternary operator instead of double check "isMarried" value
And it will be much bette use const for "wife" and "husband" 👍